aws-rds-customdbengineversion/aws-rds-customdbengineversion.json

```diff
@@ -103,12 +115,13 @@
     "/properties/KMSKeyId": "$join([\"arn:(aws)[-]{0,1}[a-z]{0,2}[-]{0,1}[a-z]{0,3}:kms:[a-z]{2}[-]{1}[a-z]{3,10}[-]{0,1}[a-z]{0,10}[-]{1}[1-3]{1}:[0-9]{12}[:]{1}key\\/\", KMSKeyId])"
   },
   "required": [
-    "DatabaseInstallationFilesS3BucketName",
     "Engine",
     "EngineVersion"
   ],
   "writeOnlyProperties": [
-    "/properties/Manifest"
+    "/properties/Manifest",
+    "/properties/SourceCustomDbEngineVersionIdentifier",
+    "/properties/UseAwsProvidedLatestImage"
```

Comment:
The parameter `UseAwsProvidedLatestImage` is conditional create only. It should not be taken into account if the current CEV Image matches the latest image. This parameter comes into play when there is a newer image available. In order to avoid confusion, we should consider making some design decisions regarding this parameter. One option could be renaming it to `UseAwsProvidedLatestImageAtCreate` to clarify its purpose or making it intelligent enough to automatically upgrade the engine when a newer image is available. Alternatively, we could explore the possibility of specifying two separate flags, one for creation and one for update. Ultimately, the solution we choose should make the most sense from the customer's perspective.